### PR TITLE
just change route to '/' + key on import, forcing server render

### LINKED ISSUE #243: import dat link input: should use router instead of webrtc to render contents

### PR DESCRIPTION
**Pull request by karissa**
fixes #243

### CODE REVIEW

File: client/js/models/archive.js [hunk 1]
```diff
@@ -130,11 +130,7 @@ module.exports = {
       send('archive:initImportQueue', {archive}, noop)
     },
     import: function (data, state, send, done) {
-      const location = '/' + data
-      send('location:setLocation', { location }, done)
-      window.history.pushState({}, null, location)
-      send('archive:update', {entries: {}, numPeers: 0, downloadTotal: 0, uploadTotal: 0, size: 0}, noop)
-      send('archive:load', data, done)
```
Comment (juliangruber) [inline client/js/models/archive.js]: shouldn't those two events be triggered by the location change already?

Comment (karissa) [inline client/js/models/archive.js]: we actually need to change the location and trigger a server-render, which this was not doing before (it was just changing the url)

Comment (juliangruber) [inline client/js/models/archive.js]: ah gotcha, then +1 (unless there's a clean way to do this without the server render)

**Comment by karissa** [comment]
good catch @juliangruber thanks1

**Pull request merged**